R/foreign.R

```diff
@@ -596,12 +596,13 @@ write.graph.dot <- function(graph, file, ...) {
 #' @family foreign
 #' @export
 #' @keywords graphs
-graph_from_graphdb <- function(url = NULL,
-                               prefix = "iso", type = "r001", nodes = NULL, pair = "A", which = 0,
-                               base = "http://cneurocvs.rmki.kfki.hu/graphdb/gzip",
-                               compressed = TRUE, directed = TRUE) {
+graph_from_graphdb <- function(
+    url = NULL,
+    prefix = "iso", type = "r001", nodes = NULL, pair = "A", which = 0,
+    base = "https://github.com/schochastics/graphsdb/raw/refs/heads/main",
```

Comment:
could we instead use a permalink from the Zenodo record?

Reply:
The zenodo link is unfortunately the whole db. You cannot just download a single file which is need for the function

Reply:
Oh ok, sorry, makes sense, thank you.

Reply:
Would we be allowed to store a copy in https://github.com/igraph/igraphdata?

Reply:
can confirm, the function works with a local copy of the database